spec/support/api_spec_helper.rb

```diff
@@ -16,19 +16,8 @@ module ApiSpecHelper
     "Accept"       => "application/json"
   }
 
-  API_STATUS = Rack::Utils::HTTP_STATUS_CODES.merge(0 => "Network Connection Error")
-
-  def parse_response
-    @code    = response.status
-    @result  = case @code
-               when Rack::Utils.status_code(:no_content)   then {}
-               when Rack::Utils.status_code(:unauthorized) then response.body
-               else JSON.parse(response.body)
-               end
-    @success = response.success?
-    @status  = API_STATUS[@code] || (@success ? Rack::Utils.status_code(:ok) : Rack::Utils.status_code(:bad_request))
-    @message = @result.kind_of?(Hash) ? @result.fetch_path("error", "message").to_s : @result
-    @success
+  def response_hash
+    JSON.parse(response.body)
```

Comment:
I thought about that, but was in two minds. If I do I'll be enabling a test smell, i.e. by making multiple expectations on the response body. We might be doing that in some places, but I'd like to follow that up in a future PR.